profiles.yaml

```diff
@@ -0,0 +1,58 @@
+# yaml-language-server: $schema=https://creativeprojects.github.io/resticprofile/jsonschema/config-1.json
+
+version: "1"
+
+groups:
+  ddev:
+    - ddev-db
+    - ddev-files
+
+default:
+  repository: "s3:blr1.digitaloceanspaces.com/axl-backup-contrib-tracker/{{ .Profile.Name }}"
+  insecure-no-password: true
+  backup:
+    source-relative: true
+
+  run-before:
+    - 'echo "AWS_ACCESS_KEY_ID=$(op read "op://Platform Engineering/Contrib Tracker S3 Backup/username")" >> "{{env}}"'
+    - 'echo "AWS_SECRET_ACCESS_KEY=$(op read "op://Platform Engineering/Contrib Tracker S3 Backup/credential")" >> "{{env}}"'
+
+  env:
+    TMPDIR: /tmp
+
+# We need to have two profiles: one for db and other for files.
+# This is because the paths are different for both.
+# And we want to use `delete` with the files profile.
+# Setting `delete` on a base path will delete everything
+# in that directory. The common base directory here is
+# the project directory and using `delete` for this will
+# delete all the source files, not just the ones backed up.
+ddev-db:
+  inherit: default
+  backup:
+    run-before:
+      - mkdir -p .ddev/.downloads
+      - ddev export-db --file=.ddev/.downloads/db-{{ .Profile.Name }}.sql.gz
+    run-after:
+      - rm .ddev/.downloads/db-{{ .Profile.Name }}.sql.gz
+    source-base: "{{ .ConfigDir }}"
+    source:
+      - ".ddev/.downloads/db-{{ .Profile.Name }}.sql.gz"
+
+  restore:
+    target: "{{ .ConfigDir }}"
+    run-after:
+      - ddev import-db --file=.ddev/.downloads/db-{{ .Profile.Name }}.sql.gz
+      - rm .ddev/.downloads/db-{{ .Profile.Name }}.sql.gz
+
+ddev-files:
```

Comment:
@hussainweb Do we need the file backups as well if we use [Stage file proxy](https://www.drupal.org/project/stage_file_proxy)?

Reply:
Please see my comment about maintaining backups anyway, even if we don't use it locally. If you don't want to restore files for any reason (like when using SFP), then you can just restore the database with `resticprofile ddev-db.restore latest`.

Reply:
ok. I was wondering if we are not going to use files that frequently due to above reason they usually take a lot more size in S3 which also increases the cost.